docs/developer-guide/syntax-reference.rst

```diff
@@ -1124,6 +1124,34 @@ SHOW PROPERTIES
 List the :ref:`configuration settings <ksql-param-reference>` that are
 currently in effect.
 
+SET/UNSET property
+------------------
+**Synopsis**
+
+.. code:: sql
+
+    SET 'auto.offset.reset' = 'earliest';
```

Comment:
```suggestion
    [SET/UNSET] 'property_name' = 'property_value';
```
If we want an example, then we can have that below (this follows the rest of the syntax doc)

Reply:
+1 to the example idea.

Reply:
Made the change.